javascript-loops/main.js

```diff
@@ -0,0 +1,75 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+
+function getNumbersToTen(){
+    let numbers = [];
+    let currentNumber = 1;
+
+    while (currentNumber <= 10){
+        numbers.push(currentNumber);
+        currentNumber++;
+    }
+    return numbers;
+};
+
+function getEvenNumbersToTwenty(){
+    let evenNumbers = [];
+    let currentNumber = 2;
+    
+    while (currentNumber <= 20){
+        evenNumbers.push(currentNumber);
+        currentNumber+=2;
+    }
+    return evenNumbers;
+};
+
+function repeatWord(word, times){
+    let count = 1;
+    let repeated = '';
+
+    while (count <= times){
+        repeated += word;
+        count++;
+    }
+    return repeated;
+};
+
+function logEachCharacter(string){
+
+    for (let i = 0; i < string.length; i++){
+        console.log(string[i]);
+    }
+};
+
+function doubleAll(numbers){
+    let doubled = [];
+    for (let i = 0; i < numbers.length; i++){
+        doubled.push(numbers[i] * 2);
+    }
+    return doubled;
+};
+
+function getKeys(object){
+    let keys = [];
+    for (key in object){
+        keys.push(key)
+    }
+    return keys;
+};
+
+function getValues(object){
+    let values = [];
+    for (let key in object){
+        if (object.hasOwnProperty(key)){
```

Comment:
`hasOwnProperty` isn't needed. We're looping over the properties that exist in the object, so all of these will exist.